src/pages/workspaces/workspace/analysis/modals/AnalysisModal.ts

```diff
@@ -415,9 +417,9 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
         title: 'Select an application',
         titleStyles: { margin: '1.5rem 0 0 1.5rem', display: !!viewMode ? 'none' : undefined },
         style: { width },
-        titleExtras: [],
+        titleChildren: [],
         onDismiss,
-        onPrevious: !!viewMode ? () => resetView() : undefined
+        onPrevious: () => !!viewMode ? () => resetView() : undefined
```

Comment:
this isn't correct, resetView will never be called here, as it is `() => () => resetView`